test-network-function/testconfigure.yml

```diff
@@ -16,5 +16,4 @@ operatortest:
   - name: "OPERATOR_STATUS"
     tests:
       - "CSV_INSTALLED"
-      - "SUBSCRIPTION_INSTALLED"
```

Comment:
was the subscription_installed test removed on purpose ?

Reply:
Yes. Close to the end of release 2.0, Ryan created a separate test and removed it from this combo test. The test identifier and this yaml were missed. I updated the identifier in my last PR. Just found out this while going through all of our yml files.